app/templates/questionnaire.html

```diff
@@ -63,10 +63,11 @@ <h1 class="alert__title venus">This page has {{all_errors|length}} errors</h1>
 
       <button class="btn btn--primary qa-btn-submit venus" type="submit" name="action[save_continue]">{% block submit_button_text %}Save and continue{% endblock %}</button>
       <br/>
+      {% if state.group_instance > 0 or not block.first_in_schema() %}
```

Comment:
Except the navigation PR hides the previous link for the first block in a group; so I guess this will conflict (or merge out) when that lands?